apps/package.json

```diff
@@ -133,6 +133,7 @@
     "mocha": "^3.1.2",
     "mochify": "2.18.0",
     "mockfirebase": "^0.12.0",
+    "mock-firmata": "0.1.0",
```

Comment:
Rick Waldron's [mock-firmata](https://github.com/rwaldron/mock-firmata), which gets [used in the johnny-five unit tests](https://github.com/rwaldron/johnny-five/blob/master/test/common/bootstrap.js#L96).